File: examples/linnea/GLS/experiments/1000_230/variants/Matlab/operand_generator.m
function [out] = operand_generator()
    import MatrixGenerator.*;
    out{ 1 } = generate([1000,230], Shape.General(), Properties.Random([-1, 1]));
    out{ 2 } = generate([1000,1000], Shape.Symmetric(), Properties.SPD());
    out{ 3 } = generate([1000,1], Shape.General(), Properties.Random([-1, 1]));
end

File: examples/linnea/GLS/experiments/1000_230/variants/Julia/operand_generator.jl
using MatrixGenerator

function operand_generator()
    X::Array{Float64,2} = generate((1000,230), [Shape.General, Properties.Random(-1, 1)])
    M::Symmetric{Float64,Array{Float64,2}} = generate((1000,1000), [Shape.Symmetric, Properties.SPD])
    y::Array{Float64,1} = generate((1000,1), [Shape.General, Properties.Random(-1, 1)])
    return (X, M, y,)
end